Cargo.toml

```diff
@@ -90,9 +90,13 @@ static_assertions = "1.1"
 thiserror = "1.0"
 tokio = "1.0"
 
+[profile.dev]
+panic = "abort"
+
 [profile.release]
 debug-assertions = true
 overflow-checks = true
+panic = "abort"
 
 [profile.test]
```

Comment:
Should the same be done for the tests?

Reply:
> Should the same be done for the tests?

Probably not, otherwise a single failed test would abort all tests.

Reply:
I haven't thought it through. 🙃 Also I have just found out that the `panic` setting is simply ignored for test, bench and dev profiles.

Reply:
It's not ignored for `dev`. I just checked it again.

Reply:
Yes, you are right. It is only ignored for test and bench profiles.